Makefile

```diff
@@ -1,5 +1,6 @@
 # Makefile to rebuild MK64 split image
 
+
```

Comment:
why add a new-line fix typos ?

Reply:
The first PR's of this account were introducing curl script download, potential malware into repos and now for whatever reason is spamming a bunch of repositories with single new empty line change PRs

Reply:
ok ok